tst/Codelyzer.Analysis.Tests/Constants.cs

```diff
@@ -5,5 +5,13 @@ internal class Constants
         // Do not change these values without updating the corresponding line in .gitignore:  **/Projects/Temp
         // This is to prevent test projects from being picked up in git after failed unit tests.
         internal static readonly string[] TempProjectDirectories = { "Projects", "Temp" };
+
+        internal static string programFiles = "Program Files";
+        internal static string programFilesx86 = "Program Files (x86)";
+        internal static string vs2022MSBuildPath = @"Program Files\Microsoft Visual Studio\2022\Professional\MSBuild\Current\Bin\MSBuild.exe";
+        internal static string vs2019MSBuildPath = @"Program Files (x86)\Microsoft Visual Studio\2019\Professional\MSBuild\Current\Bin\MSBuild.exe";
+        internal static string vs2017MSBuildPath = @"Program Files (x86)\Microsoft Visual Studio\2017\Professional\MSBuild\15.0\Bin\MSBuild.exe";
+        internal static string vs2019BuildToolsMSBuildPath = @"Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\MSBuild\Current\Bin\MSBuild.exe";
```

Comment:
Why do we need to add these? Aren't we looking in the folder structure anyway?

Reply:
These are the expected values for test cases. I put them under constants. What we look and get are the actual values.